.github/workflows/workflow.yml

```diff
@@ -0,0 +1,37 @@
+name: aws-api-test
+
+on:
+  pull_request:
+    branches: ['develop-add-tests'] # for testing purpose, added develop-add-tests
+  push:
+    branches: ['develop', 'develop-add-tests'] # for testing purpose, added develop-add-tests
+  workflow_dispatch: # allow workflow to be manually triggered
+
+
+jobs:
+  Benchmark-Latency:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Activate a virtual environment 
+        run: |
+          pip install --upgrade pip
+          python3.10 -m venv venv
+          source venv/bin/activate
+
+      - name: Install dependencies
+        run: pip3 install -r requirements.txt
+
+      - name: Generate manifest test
+        env:
+          SYNAPSE_ACCESS_TOKEN: ${{ secrets.SYNAPSE_ACCESS_TOKEN }}
+          SERVICE_ACCOUNT_CREDS: ${{ secrets.SERVICE_ACCOUNT_CREDS }}
```

Comment:
The synapse python client actually automatically detects the `SYNAPSE_AUTH_TOKEN` environment variable.  I would rename the `SYNAPSE_ACCESS_TOKEN` and remove `SERVICE_ACCOUNT_CREDS`

Reply:
Thanks Tom. I have updated the code to use SYNAPSE_AUTH_TOKEN